### PR TITLE
Corrected style classes for spree >= 4.0

### PR DESCRIPTION
**Pull request by voleoo**
Current gem should have dependency `coffee-rails` because uses CoffeeScript.
Generally new apps do not uses 'coffee' and we had strange errors which should do not have.

### CODE REVIEW

File: app/overrides/add_documents_to_main_navigation.rb [hunk 1]
```diff
@@ -1,6 +1,15 @@
-Deface::Override.new(
-  virtual_path:  'spree/layouts/admin',
-  insert_bottom: '#main-sidebar',
-  partial:       'spree/admin/shared/menu/documents_tab',
-  name:          'documents_tab'
-)
+if Spree.version.to_f < 4.0
```
[the remark this answers is not in the document]
Comment (voleoo) [reply app/overrides/add_documents_to_main_navigation.rb:3]: > # frozen_string_literal: true

fixed

**Comment by imgarylai** [comment]
I am working on this problem as well. 

I would like to propose just rewrite this piece of code in Javascript(this is my workaround). And we do not have to add another run time dependency anymore.

**Comment by voleoo** [comment]
> I am working on this problem as well.
> 
> I would like to propose just rewrite this piece of code in Javascript(this is my workaround). And we do not have to add another run time dependency anymore.

I think it's good idea for rewrite 5 lines of `CoffeeScript` to JS and remove dependency.
@damianlegawiec what do you think?

**Comment by damianlegawiec** [comment]
> > I am working on this problem as well.
> > I would like to propose just rewrite this piece of code in Javascript(this is my workaround). And we do not have to add another run time dependency anymore.
> 
> I think it's good idea for rewrite 5 lines of `CoffeeScript` to JS and remove dependency.
> @damianlegawiec what do you think?

@voleoo @imgarylai 

Yes, please rewrite it in JS, you can use https://michaelsboost.com/Coffee2JS/ converter. We dropped Coffee support around Spree 4.0

Thank you all!

**Comment by voleoo** [comment]
> > > I am working on this problem as well.
> > > I would like to propose just rewrite this piece of code in Javascript(this is my workaround). And we do not have to add another run time dependency anymore.
> > 
> > 
> > I think it's good idea for rewrite 5 lines of `CoffeeScript` to JS and remove dependency.
> > @damianlegawiec what do you think?
> 
> @voleoo @imgarylai
> 
> Yes, please rewrite it in JS, you can use https://michaelsboost.com/Coffee2JS/ converter. We dropped Coffee support around Spree 4.0
> 
> Thank you all!

Dependency removed. @damianlegawiec could you look at PR and marge it. Thanks.

**Comment by voleoo** [comment]
@damianlegawiec please do not merge it, I have checked it and still need resolve issue with dependency.
I'll message you.

**Comment by imgarylai** [comment]
@voleoo I am also working on to make this gem work with Spree >= 4. I create a PR on #139 Part of the PR is similar to this one. Would you mind taking a look at it? 

We might be able to work together and make everything work. 

Thanks!

**Comment by voleoo** [comment]
> @voleoo I am also working on to make this gem work with Spree >= 4. I create a PR on #139 Part of the PR is similar to this one. Would you mind taking a look at it?
> 
> We can work together and make everything work.
> 
> Thanks!

Yes, we can work together. All community works together.
I have some comments to your PR #139 

Could you separate your PR to some branches.
Locales, new feature, fixes, refactoring existing code etc.
It's for better understanding what is going on.

Thanks!

**Comment by imgarylai** [comment]
@voleoo yes, sure! Sorry for this huge PR. I will resubmit a new PR again.

**Pull request closed without merging**